internal/services/network/subnet_resource.go

```diff
@@ -201,6 +201,13 @@ func resourceSubnet() *pluginsdk.Resource {
 				},
 			},
 
+			"default_outbound_access_enabled": {
+				Type:     pluginsdk.TypeBool,
+				Computed: true,
```

Comment:
Whilst it's changing in the future, since at this point-in-time this defaults to `true` we should match that on our side:

```suggestion
				Default: true,
```